plugins/obs-qsv11/obs-qsv-test/obs-qsv-test.cpp

```diff
@@ -27,17 +27,11 @@ struct adapter_caps {
 	bool supports_hevc = false;
 };
 
-static std::vector<uint64_t> luid_order;
```

Comment:
Why are the LUID code sections removed? IIRC, we required them because OBS and the test executable could end up on different GPUs, so the GPU enumeration order could be different between OBS and the test executable. We used the LUIDs to ensure that the adapter capabilities test results were mapped to the correct adapter. Has something changed where this is no longer a problem?

Reply:
Sorry, I was unaware of this, I added back the LUID code sections.